### ISSUE TITLE
FontSize

### ISSUE THREAD

**Issue by GitOfHaiyang** [billryan/resume#63]
Could I adjust my font size? It looks so larger that I can not put all the word in one page

**Comment by v1siuol** [comment]
YES. `resume.cls` is the primary file that controls the structure, format and layout of the resume. 

To adjust the font size, try to edit on here (e.g. `\LoadClass[10pt]{article}`) https://github.com/billryan/resume/blob/37ffacefb7bd0cda9dd490371a054bfebd0929e9/resume.cls#L3

**Comment by lisihang** [comment]
请问一下，为什么修改了之后大小还是不变呢

**Comment by Chauncey-Wang** [comment]
> 

请问您解决了吗？

**Comment by Chauncey-Wang** [comment]
> 请问一下，为什么修改了之后大小还是不变呢

请问您解决了吗？？

**Comment by GitOfHaiyang** [comment]
> > 请问一下，为什么修改了之后大小还是不变呢
> 
> 请问您解决了吗？？

解决了，你按照上面的回答就能改字体
